src/Cli/dotnet/Commands/Package/Add/PackageAddCommandParser.cs

```diff
@@ -18,7 +16,7 @@ public static class PackageAddCommandParser
     .AddCompletions((context) =>
     {
         // we should take --prerelease flags into account for version completion
-        var allowPrerelease = context.ParseResult.GetValue(PrereleaseOption);
+        var allowPrerelease = context.ParseResult.GetValue(PrereleaseOption!);
```

Comment:
It looks like the order of the declarations is the only reason the suppressions are needed. That seems unfortunate.